datastore/src/test/java/io/spine/server/storage/datastore/NewBoundedContextBuilderTest.java

```diff
@@ -45,14 +43,14 @@ void testProduceBCBuilder() {
         DatastoreStorageFactory factory = givenFactory();
         BoundedContextBuilder builder = BoundedContext.multitenant(
                 NewBoundedContextBuilderTest.class.getName());
-        assertFalse(builder.tenantIndex()
-                           .isPresent());
+        assertThat(builder.tenantIndex()).isEmpty();
 
         Optional<? extends TenantIndex> updatedIndex =
                 factory.configureTenantIndex(builder)
                        .tenantIndex();
-        assertTrue(updatedIndex.isPresent());
-        assertThat(updatedIndex.get(), instanceOf(TestNamespaceIndex.getType()));
+        assertThat(updatedIndex).isPresent();
+        Truth.assertThat(updatedIndex.get())
```

Comment:
Please use a static import. Java can handle multiple imported methods with the same name if parameters are different.

Reply:
What's interesting is that IDEA does not allow fixing if not doing the static import manually.